packages/lwc/scripts/build.js

```diff
@@ -13,23 +13,11 @@ const distDirectory = path.join(__dirname, '../dist');
 
 const COMMON_TARGETS = [
     // ESM
-    { target: 'es2017', format: 'esm', prod: false },
+    { target: 'es2017', format: 'esm' },
 
     // IIFE
-    { target: 'es5', format: 'iife', prod: false },
-    { target: 'es5', format: 'iife', prod: true },
-    { target: 'es5', format: 'iife', prod: true, debug: true },
-    { target: 'es2017', format: 'iife', prod: false },
-    { target: 'es2017', format: 'iife', prod: true },
-    { target: 'es2017', format: 'iife', prod: true, debug: true },
-
-    // UMD
-    { target: 'es5', format: 'umd', prod: false },
-    { target: 'es5', format: 'umd', prod: true },
-    { target: 'es5', format: 'umd', prod: true, debug: true },
-    { target: 'es2017', format: 'umd', prod: false },
-    { target: 'es2017', format: 'umd', prod: true },
-    { target: 'es2017', format: 'umd', prod: true, debug: true },
+    { target: 'es5', format: 'iife' },
+    { target: 'es2017', format: 'iife' },
```

Comment:
I find UMD a bit harder to read, and also it's not supported by ESBuild if we ever decide to use it (https://github.com/evanw/esbuild/issues/507).

What if we get rid of both IIFE and UMD, and use our own custom bundling in the Karma tests? I only used IIFE to avoid breaking those tests.

Reply:
I am fine removing IIFE and UMD, and using a build step for Karma and Webdriver tests.